.github/workflows/pr-build.yml

```
@@ -30,6 +30,62 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v2
 
+      - name: Build smoke test image
+        uses: docker/build-push-action@v3
```

actionlint (1.7.12)

[error] the runner of "docker/build-push-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)